app/scripts/modules/core/application/modal/applicationProviderFields.component.js

```diff
@@ -14,8 +16,17 @@ module.exports = angular
       },
       controller: 'ApplicationProviderFieldsCtrl',
   })
-  .controller('ApplicationProviderFieldsCtrl', function(cloudProviderRegistry) {
+  .controller('ApplicationProviderFieldsCtrl', function(cloudProviderRegistry, settings, _) {
     const templateUrlPath = 'applicationProviderFields.templateUrl';
+    let defaultProviderFields = settings.providers;
+
+    this.initializeApplicationFields = (fieldPath) => {
```

Comment:
`initializeApplicationFields` implies that multiple fields can be set by this method. It should either take a spread operator (`this.initializeApplicationFields = (...fieldPath) => `, then iterate over them) or be renamed to `initializeApplicationField`.